site/_posts/2017-04-07-blood-fire-manageiq.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Blood, fire and ManageIQ"
+author: panspagetka
+date: 2017-04-07
```

Comment:
Probably not a big deal, but you may want to make this 04-11, the date when we aim to publish